src/main/webapp/app/shared/feature-toggle/feature-toggle.directive.ts

```diff
@@ -30,17 +30,23 @@ export class FeatureToggleDirective implements OnInit, OnDestroy {
      */
     ngOnInit() {
         // If no feature is set for the toggle, the directive does nothing.
-        if (this.feature) {
-            this.featureToggleActiveSubscription = this.featureToggleService
-                .getFeatureToggleActive(this.feature)
-                .pipe(
-                    // Disable the element if the feature is inactive.
-                    tap((active) => {
-                        this.featureActive = this.skipFeatureToggle || active;
-                    }),
-                )
-                .subscribe();
+        if (!this.features) {
```

Comment:
This method is duplicated in `feature-toggle.directive.ts` and `feature-toggle-link.directive.ts`. Can you find a way to remove this duplication?

Reply:
The whole directive is basically a duplicate. Actually, editing the toggle link directive isn't in scope of this PR as I'm not using it anywhere in this PR, so I've just removed the modifications on this file.

We can try to combine both directives in general later, not only that single method, but it's out of scope here